cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -1966,11 +1966,8 @@ GfErrType ThinClientPoolDM::sendRequestToEP(const TcrMessage& request,
 }
 
 TcrEndpoint* ThinClientPoolDM::addEP(ServerLocation& serverLoc) {
-  std::string serverName = serverLoc.getServerName();
-  int port = serverLoc.getPort();
-  char endpointName[100];
-  std::snprintf(endpointName, 100, "%s:%d", serverName.c_str(), port);
-
+  std::string endpointName = serverLoc.getServerName() + ":"
```

Comment:
Please use `const auto`, `const auto&` or `auto` whenever possible.

Reply:
Thanks, I have fixed it.